src/Backend/Modules/Pages/Engine/Model.php

```diff
@@ -89,8 +89,7 @@ public static function buildCache(string $language = null): void
 
     public static function copy(string $fromLanguage, string $toLanguage): void
     {
-        $copyModulesToOtherLocaleManager = BackendModel::getContainer()->get('fork.manager.copy_modules_to_other_locale');
-        $copyModulesToOtherLocaleManager->copy(
+        BackendModel::getContainer()->get('fork.manager.copy_modules_to_other_locale')->copy(
```

Comment:
now also remove the obsolete `getContainer()->`

Reply:
Fixed